### PR TITLE
fix(costmodel): address warning messages in the console log

### LINKED ISSUE #1010: Error/warning log in console in cost models page
**Describe the bug**
In development mode there are error and warning messages regard missing key/aria-label prop in some element.

**Expected behavior**
Although these messages don't mean that the system will crash immediately they do alert about a problem that will cause a failure in the future.

**Screenshots**
Screenshot from 2019-09-23 17-15-25

Screenshot from 2019-09-23 17-15-55

Screenshot from 2019-09-23 17-16-10

### PR DESCRIPTION
**Pull request by boaz0**
closes #1010

### CODE REVIEW

File: src/pages/costModelsDetails/costModelInfo/index.tsx [hunk 1]
```diff
@@ -56,6 +56,7 @@ class CostModelInformation extends React.Component<Props, State> {
           <TabContent
             eventKey={0}
             id="refPriceList"
+            title="PriceList"
```
Comment (dlabrecq) [inline src/pages/costModelsDetails/costModelInfo/index.tsx]: Not sure how `TabContent` uses titles, but should be "Price list" for a title / aria attributes?

File: src/pages/costModelsDetails/costModelInfo/header.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ class Header extends React.Component<Props> {
         >
           <Tab
             eventKey={0}
-            title="Price list"
+            title="Pricelist"
```
Comment (dlabrecq) [inline src/pages/costModelsDetails/costModelInfo/header.tsx]: This should be two words (i.e., what you had there originally).

File: src/pages/costModelsDetails/costModelInfo/index.tsx (hunk 1 above)
Comment (dlabrecq) [reply src/pages/costModelsDetails/costModelInfo/index.tsx]: Actually, I don't see `title` as a valid property for `TabContent`. Should be able to just remove it?

**Comment by boaz0** [comment]
@dlabrecq thanks for the review - fixed all the comments.

also closes #1034

**Comment by boaz0** [comment]
Thanks @dlabrecq

**Pull request merged**